boards/arm/nucleo_f091rc/nucleo_f091rc.dts

```diff
@@ -40,3 +40,11 @@
 	status = "ok";
 	clock-frequency = <I2C_BITRATE_FAST>;
 };
+
+&spi1 {
+	status = "ok";
+};
+
+&spi2 {
```

Comment:
If we enable SPI2 port here, we should also enable it in Kconfig.defconfig.

Reply:
done

Reply:
I removed the node, since the corresponding pinmux defines are missing.
See https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/10601/3/tests